src/frontend/komponenter/Oppgavebenk/oppgavefilterStorage.ts

```diff
@@ -0,0 +1,18 @@
+export const oppgaveRequestKey = 'oppgaveFiltreringRequest';
+
+export const lagreTilLocalStorage = <T>(key: string, request: T): void => {
+    try {
```

Comment:
varfør try/finally her ? :P

Reply:
```setItem``` kan kaste en exception på iphone/safari. Men ingen grunn til vi bruker ```finally``` istedenfor ```catch``` hvis det var det du tenkte på?

Reply:
Truuur det kommer dröja tills saksbehandligen kommer att föregå på iphone ass :P

Reply:
Meen jag förstår inte :( Anta att saksbehandlingen föregår på iphone vad er det som catchar den felen ?

try {
  throw new Error("Saksbehandlar på iphone");
} finally { };

det her ger fortfarande fel

Reply:
Ah, du har helt rett! Jeg har misforstått forskjellen på catch og finally :(

Reply:
Visste in att setItem kan kaste en exception på iphone/safari !